src/constants/index.js

```diff
@@ -34,6 +34,7 @@ export const SSL_PORT = '443';
 
 export const GET = 'GET';
 export const POST = 'POST';
+export const PUT = 'PUT';
```

Comment:
Could you add a test case for this constant?
In https://github.com/LiskHQ/lisk-js/pull/599 I have added all the other missing constants.